common4j/src/main/com/microsoft/identity/common/java/controllers/CommandDispatcher.java

```diff
@@ -167,6 +178,60 @@ public static void submitSilent(@SuppressWarnings(WarningType.rawtype_warning) @
         submitSilentReturningFuture(command);
     }
 
+    /**
+     * submitSilentAndReturnAuthorizationResult - Run the Device Code Flow command using a DCF thread pool
+     * @param command   {@link DeviceCodeFlowAuthResultCommand}
+     * @return {@link AuthorizationResult}
+     * @throws BaseException
+     */
+    public static AuthorizationResult submitSilentAndReturnAuthorizationResult(@NonNull final DeviceCodeFlowAuthResultCommand command)
```

Comment:
>submitSilentAndReturnAuthorizationResult [](http://example.com/codeflow?start=38&length=40)

nit: rename to something like submitDcfAuthResultSilent
and submitDcfTokenResultSilent